### PR TITLE
docs: Create Support "known-issues-and-solutions"

### LINKED ISSUE #3827: Docs: add known issues and solutions section to the docs
Related to https://github.com/netlify/netlify-cms/issues/2827

We should add a "known issues" section to the docs so users can self serve common issues.
Partial list to be updated:
- [ ] GitHub private+public+organisations permissions issues (e.g. https://github.com/netlify/netlify-cms/issues/1415)
- [ ] Having a branch named `cms` fails editorial workflow (https://github.com/netlify/netlify-cms/issues/3065)
- [ ] Working with multiple CMS instances/environments (e.g https://github.com/netlify/netlify-cms/issues/3790)
- [ ] Git gateway + Identity issues (e.g. https://github.com/netlify/netlify-cms/issues/3718 and https://community.netlify.com/t/common-issue-netlify-cms-git-gateway-email-not-confirmed/10690)

Please add more in the comments if you have them

### PR DESCRIPTION
**Pull request by lilpolymath**
**Summary**
WIP fixes #3827 
 Known Issues and Solution section to help users self-serve common issues.

### CODE REVIEW

File: website/content/docs/known-issues-and-solutions.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+#### 1. Error Message.
+
+```
+API_ERROR: {"message":"404 File Not Found"}
+```
+
+#### Possible Causes.
+
+- The collection folder or file property is missing a value (a dedicated file or folder, can't be blank or root). The file or folder should be relative to root (so `src/posts` for example).
```
Comment (erezrokah) [inline website/content/docs/known-issues-and-solutions.md:23]: I think these scenarios should be fixed (we don't throw an error, but just won't list any entries).
Do you mind verifying it?

File: website/content/docs/known-issues-and-solutions.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+#### 2. Error Message
+
+```
+API_ERROR: Not Found
+```
+
+#### Possible Cause
+
+- The media library folder is missing, or the value of the media library property within config.yml isn't correct.
```
Comment (erezrokah) [inline website/content/docs/known-issues-and-solutions.md:34]: Don't think this happens anymore. Can you please verify it?

File: website/content/docs/known-issues-and-solutions.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+#### 3. Error Message
+
+```
+Failed to load entries: API_ERROR: Not Found
+```
+
+#### Possible Causes
+
+- Git Gateway is not enabled: Enable Git Gateway from the Netlify Identity settings or refresh the token.
```
Comment (erezrokah) [inline website/content/docs/known-issues-and-solutions.md:44]: - Git Gateway is not enabled - I think we show a specific error in that case. If not we should be able to fix the code here https://github.com/netlify/netlify-cms/blob/9119011c8fd304742db7490027de7faf9cec5ee0/packages/netlify-cms-backend-git-gateway/src/implementation.ts#L284. Can you check what is the error reported in that case.
- There aren't any entries for a given collection - don't think this should happen anymore

File: website/content/docs/known-issues-and-solutions.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+#### 4. Error Message
+
+```
+Failed to load entry: API_ERROR: Not Found
+```
+
+#### Possible Causes
+
+- The collection folder or file property is missing a value (a dedicated file or folder, can't be blank or root). The file or folder should be relative to root (so `src/posts` for example).
```
Comment (erezrokah) [inline website/content/docs/known-issues-and-solutions.md:55]: The first cause here is also under `1. Error Message.` and the second one shouldn't happen anymore.

File: website/content/docs/known-issues-and-solutions.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+#### 5. Error Message
+
+```
+Failed to persist entry: API_ERROR: Not Found
+```
+
+#### Possible Cause
+
+- The org account needs to grant access to your Netlify CMS instance. [More details](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/requesting-organization-approval-for-oauth-apps).
```
Comment (erezrokah) [inline website/content/docs/known-issues-and-solutions.md:66]: This is still relevant I think. Nice one!
Not sure the error is `Failed to persist entry` (can you please verify?)

File: website/content/docs/known-issues-and-solutions.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+
+#### 6. Error Message
+
+```
+Failed to persist entry: API_ERROR: Reference update failed
+```
+
+#### Possible Cause
+
+- You're using the editorial workflow but already have a branch called cms. This error is fixed by removing the cms branch from the repo.
```
Comment (erezrokah) [inline website/content/docs/known-issues-and-solutions.md:76]: We now handle this error and report it properly, see https://github.com/netlify/netlify-cms/pull/3879

File: website/content/docs/known-issues-and-solutions.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+#### 7. Error Message
+
+```
+Failed to load settings from /.netlify/identity
+```
+
+#### Possible Cuase
+
+- Netlify Identity isn't enabled.
```
Comment (erezrokah) [inline website/content/docs/known-issues-and-solutions.md:86]: We now handle this error properly, see https://github.com/netlify/netlify-cms/blob/9119011c8fd304742db7490027de7faf9cec5ee0/packages/netlify-cms-backend-git-gateway/src/AuthenticationPage.js#L97

File: website/content/docs/known-issues-and-solutions.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+
+- Change the invitation emails! This takes a bit more time, but it’s probably the cleanest solution. You already have the identity widget loaded at your `admin/` page, so you might as well have your users confirm their email there. Learn how to update your invitation emails [here](https://docs.netlify.com/visitor-access/identity/identity-generated-emails/#email-templates).
+
+#### 2. Invitation Email not Recieved.
+
+If you deployed Netlify CMS to Netlify using a template and didn't receive an invitation email after deploying the site, check your email settings to make sure that your email provider doesn't block emails from no-reply@netlify.com.
+
+#### 3. Mobile responsiveness.
+
+To make Netlify CMS responsive for usage on mobile, [here](https://gist.github.com/lilpolymath/51082130d316df3a938648569077c9330) is a temporary CSS solution to include to your `admin/index.html` below your closing `</body>` tag so it can overwrite netlify-generated css.
```
Comment (erezrokah) [inline website/content/docs/known-issues-and-solutions.md:111]: https://gist.github.com/lilpolymath/51082130d316df3a938648569077c9330 gives me 404

File: website/content/docs/known-issues-and-solutions.md [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+```
+
+#### Possible Cuase
+
+- Netlify Identity isn't enabled.
+- config.yml has git-gateway as a backend on a site that isn't hosted on Netlify.
+
+## Other Issues
+
+#### 1. Invited Users Email not Confirmed.
```
Comment (erezrokah) [inline website/content/docs/known-issues-and-solutions.md:91]: We should link to https://community.netlify.com/t/common-issue-netlify-cms-git-gateway-email-not-confirmed/10690 and credit @tomrutgers

**Comment by lilpolymath** [comment]
Thank you @erezrokah I will go through everything now.

File: website/content/docs/known-issues-and-solutions.md (hunk 1 above)
Comment (lilpolymath) [reply website/content/docs/known-issues-and-solutions.md:23]: I will verify this.

File: website/content/docs/known-issues-and-solutions.md (hunk 2 above)
Comment (lilpolymath) [reply website/content/docs/known-issues-and-solutions.md:34]: No problem I will double-check just to be sure.

**Comment by lilpolymath** [comment]
@erezrokah What do you about adding [this issue](https://github.com/netlify/netlify-cms/issues/3790#issuecomment-635920266)? A couple of changes would have to be made to the config.yml file tho.

File: website/content/docs/known-issues-and-solutions.md (hunk 9 above)
Comment (lilpolymath) [reply website/content/docs/known-issues-and-solutions.md:69]: Done!

File: website/content/docs/known-issues-and-solutions.md (hunk 1 above)
Comment (lilpolymath) [reply website/content/docs/known-issues-and-solutions.md:23]: I think you are right.

File: website/content/docs/known-issues-and-solutions.md (hunk 5 above)
Comment (lilpolymath) [reply website/content/docs/known-issues-and-solutions.md:55]: "Failed to persist entry" is not part of the error.

File: website/content/docs/known-issues-and-solutions.md (hunk 1 above)
Comment (lilpolymath) [reply website/content/docs/known-issues-and-solutions.md:23]: No, it doesn't throw an error.

**Comment by erezrokah** [comment]
> What do you think about adding #3790 [this issue](https://github.com/netlify/netlify-cms/issues/3790#issuecomment-635920266)?

I think this is a great idea!
Also, thank you so much for following up with all those changes and verifications.
Let me know when you're ready for me to do another review pass.

**Comment by erezrokah** [comment]
Closing this as stale. We can re-introduce in the future and use it as the issue references it

**Pull request closed without merging**